asyncscope.md

```diff
@@ -397,7 +409,7 @@ It is safe to drop the sender returned from `spawn_future()` without starting it
 Please note that there is a race between the completion of the given sender and the start of the returned sender.
 The race will be resolved by the `@_spawn-future-sender_@<>` state.
 
-Cancelling the returned sender, cancels `s` but does not cancel the `async_scope`.
+Cancelling the returned sender, cancels `s` but does not cancel the `async_scope`, but the cancellation would be propagated to the given sender.
```

Comment:
Like a classic music video, this sentence has one too many buts.

Reply:
> Cancelling the returned sender, cancels `s` but does not cancel the `async_scope`. The cancellation propagates to the given sender.

I'm not completely sure what that means, though. Maybe we really want to say not that `s` is cancelled, but that the parameter to `spawn_future` is cancelled? Or at least, the cancellation source passed to `s` propagates to the sender passed to `spawn_future` if we want to go into details about the implementation.

Reply:
I think I just added more confusion withe the new addition; the original phrase was fine.

Cancel returned sender -> cancel input sender (which is `s`)
Cancel returned sender != cancel `async_scope`